_docs/privacy/web-tracking-protections.md

```diff
@@ -286,6 +280,34 @@ We review user-reported Broken Site Reports daily, and work to address these usa
+
+| Platform      | Support                                                                                                         |
+| ------------- | --------------------------------------------------------------------------------------------------------------- |
+| All platforms | Infrastructure Domain categorization impacts how we build our tracker lists, which are shared across platforms. |
+
+### DuckDuckGo Private Search Ads
+
+Advertising on DuckDuckGo is done in partnership with Microsoft. Viewing ads on DuckDuckGo is anonymous, and Microsoft has [committed][ads-by-microsoft] to not profile our users on ad clicks: “when you click on a Microsoft-provided ad that appears on DuckDuckGo, Microsoft Advertising does not associate your ad-click behavior with a user profile. It also does not store or share that information other than for accounting purposes.”
+
+Previously, we were limited in how we could apply our 3rd-Party Tracker Loading Protection on Microsoft tracking scripts due to a policy requirement related to our use of Bing as a source for our private search results. This is no longer the case and as of October 2022 we block Microsoft scripts from loading on 3rd-party websites in our browsing apps (MacOS, Windows, iOS, and Android) and in our browser extensions (Chrome, Firefox, Safari, Edge and Opera). To be clear, Microsoft scripts were never embedded in our search engine or apps, which do not track you. Companies insert these scripts on their own websites for their own purposes, and so they never sent any information to DuckDuckGo.
+
+To evaluate whether an ad on DuckDuckGo is effective, advertisers want to know if their ad clicks turn into purchases (conversions). To see this within Microsoft Advertising, they use Microsoft scripts from the bat.bing.com domain. Currently, if an advertiser wants to detect conversions for their own ads that are shown on DuckDuckGo, 3rd-Party Tracker Loading Protection will not block bat.bing.com requests from loading on the advertiser’s website following DuckDuckGo ad clicks and 1st-Party Cookie Protection will expire any cookies set by these requests on the advertiser’s website after 7 days, but these requests are blocked in all other contexts. For anyone who wants to avoid this, it's possible to disable ads in DuckDuckGo search settings. And in any case, our other overlapping protections — including 3rd-Party Cookie Protection and Fingerprinting Protection — continue to apply, even in the context of DuckDuckGo ad clicks.
```

Comment:
```suggestion
To evaluate whether an ad on DuckDuckGo is effective, advertisers want to know if their ad clicks turn into purchases (conversions). To see this within Microsoft Advertising, they use Microsoft scripts from the `bat.bing.com` domain. Currently, if an advertiser wants to detect conversions for their own ads that are shown on DuckDuckGo, 3rd-Party Tracker Loading Protection will not block `bat.bing.com` requests from loading on the advertiser’s website following DuckDuckGo ad clicks and 1st-Party Cookie Protection will expire any cookies set by these requests on the advertiser’s website after 7 days, but these requests are blocked in all other contexts. For anyone who wants to avoid this, it's possible to disable ads in DuckDuckGo search settings. And in any case, our other overlapping protections — including 3rd-Party Cookie Protection and Fingerprinting Protection — continue to apply, even in the context of DuckDuckGo ad clicks.
```